test/TestApps/AppLifecycleTestApp/AppLifecycleTestApp.vcxproj

```diff
@@ -42,7 +42,7 @@
     <RootNamespace>AppLifecycleTestApp</RootNamespace>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
     <ProjectSubType>NativeUnitTestProject</ProjectSubType>
-    <ProjectName>AppLifecycleTestApp</ProjectName>
+    <ProjectName>WPNWin32MSIX</ProjectName>
```

Comment:
Why replacing? Doesn't AppLifecycle have its own test app? Seems like are removing the reference in favor of our test app, which is not good.

Reply:
I don't know how it got removed but I am putting it back in the repo.